ocfweb/docs/docs/staff/backend/internal-firewalls.md

```diff
@@ -0,0 +1,126 @@
+[[!meta title="Internal firewalls"]]
+
+While the [[external firewall|doc staff/backend/firewall]] regulates network
+traffic to the OCF from outside the OCF network, internal firewalls are
+responsible for regulating network traffic between different machines within the
+OCF.
+
+OCF machines are broadly classed into internal and [DMZ][dmz]. Internal machines
+are those which are not running user code and are not staff VMs, specifically
+those in the IP range 5-90 (excluding the ones in the DMZ listed below). The DMZ
+consists of all other machines, including:
+
+ * Those running user code (tsunami, werewolves, death, and desktops)
```

Comment:
I think it would be better to avoid server names here, especially for e.g. werewolves which may get renamed.

tsunami and death may be okay since they're unlikely to change.

Reply:
I agree, werewolves is being renamed quite soon in fact.